src/main/java/com/toofifty/easyblastfurnace/methods/MetalBarMethod.java

```diff
@@ -64,7 +64,7 @@ public MethodStep next(BlastFurnaceState state)
 
         if (state.getBank().isOpen()) {
             if (state.getInventory().has(barItem(), oreItem())) {
-                return depositInventory;
+                return useDepositInventory ? depositInventory : depositBarsAndOres;
```

Comment:
Maybe chuck `useDepositInventory` in the `PlayerState` or similar, means it doesn't have to be passed around. Could also inject the config into the base state and use that

```suggestion
                return state.getPlayer().useDepositInventory() ? depositInventory : depositBarsAndOres;
```

Reply:
I've done as you asked here, made a `state.getConfig()` method. Thanks, wasn't sure how to get the config into the method classes!